src/utils/UUID.ts

```diff
@@ -0,0 +1,24 @@
+import { useStorage } from "../hooks/useStorage";
+
+interface UrlMap {
+  url: string;
+  uuid: string;
+}
```

Comment:
We can remove this since we are not using it in this file

Reply:
okay